<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <title>HARMONIES | Microbiome Network Inference | QBRC | UT Southwesetern | Southern Methodist University</title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
    <meta name="description" content=""/>
    <meta name="keywords" content=""/>

    <link rel="shortcut icon" href="images/utsw-icon.png">
    <!-- css -->
    <link href="css/bootstrap.min.css" rel="stylesheet"/>
    <link href="css/style.css" rel="stylesheet"/>

    <!-- Theme skin -->
    <link id="t-colors" href="skins/default.css" rel="stylesheet"/>

    <!-- boxed bg -->
    <link id="bodybg" href="bodybg/bg1.css" rel="stylesheet" type="text/css"/>
</head>
<body>
<div id="wrapper">
    <!-- start header -->
    <?php include("menu.php") ?>
    <!-- end header -->

    <section id="content">
        <div class="container">
            <div class="row">
                <div class="col-lg-9">
                    <article id="navdescription">
                        <div class="post-image">
                            <div class="post-heading">
                                <h3>Description</h3>
                            </div>
                        </div>
                        <p>
                            This package serves to infer the network structure of the microbiome count data generated by
                            next-generation sequencing (NGS) technology. It also provides functions to generate
                            overdispersed and zero-inflated count data from a Dirichlet Multinomial model with flexible
                            parameter settings. The simulated count data also preserve a pre-specified network structure
                            under the Gaussian graphical model.
                        </p>
                    </article>

                    <article id="navexample">
                        <div class="post-slider">
                            <div class="post-heading">
                                <h3>Example</h3>
                            </div>
                        </div>
                        <div class="code">
                            <p>
                                install.packages("devtools")<br>
                                library(devtools)<br>
                                install_github("shuangj00/HARMONIES", subdir = “pkg”)<br>

                                example.result = HARMONIES (count.matrix, phenotype, N.mcmc = 100, b = 1, h = 20,
                                sparsity.cutoff = 0.5, beta.stars = 0.05, n.rep = 20, bayes.fdr = 0.05, seed = 123)
                            </p>
                        </div>
                    </article>


                    <article id="navarguments">
                        <div class="post-video">
                            <div class="post-heading">
                                <h3>Arguments</h3>
                            </div>
                        </div>
                        <div>
                            <table class="table table-hover">
                                <tbody>
                                <tr class="table-light">
                                    <td>count.matrix</td>
                                    <td>A count matrix from metagenomic shotgun sequencing or 16SrRNA sequencing
                                        technologies. Columns represent the taxa from the same taxonomic level, and rows
                                        represent the samples
                                    </td>
                                </tr>
                                <tr class="table-active">
                                    <td>phenotype</td>
                                    <td>A phenotype vector for all the samples. It should be a vector of 0s if only have
                                        one phenotype, or a vector of 0 and 1 if have 2 phenotypes
                                    </td>
                                </tr>
                                <tr class="table-light">
                                    <td>N.mcmc</td>
                                    <td>Number of MCMC iterations
                                    </td>
                                </tr>
                                <tr class="table-active">
                                    <td>b</td>
                                    <td>Shape parameter used in the hyper-parameter settings of the variance term
                                    </td>
                                </tr>
                                <tr class="table-light">
                                    <td>h</td>
                                    <td>Scale parameter used in the hyper-parameter settings of the variance term
                                    </td>
                                </tr>
                                <tr class="table-active">
                                    <td>sparsity.cutoff</td>
                                    <td>A threshold between 0-1. Taxa with proportions of zeros larger than the
                                        threshold would be dropped for the network inference
                                    </td>
                                </tr>
                                <tr class="table-light">
                                    <td>beta.stars</td>
                                    <td>Stability threshold for selection criteria
                                    </td>
                                </tr>
                                <tr class="table-active">
                                    <td>n.rep</td>
                                    <td>Number of random subsamples to take for network estimation
                                    </td>
                                </tr>
                                <tr class="table-light">
                                    <td>bayes.fdr</td>
                                    <td>Bayesian false discovery rate controlled for zero imputation</td>
                                </tr>
                                <tr class="table-light">
                                    <td>seed</td>
                                    <td>Random seed</td>
                                </tr>
                                </tbody>
                            </table>
                        </div>
                    </article>

                    <div class="clear"></div>
                </div>
                <div class="col-lg-3">
                    <aside class="right-sidebar">
                        <div class="widget">
                            <h5 class="widgetheading">User Guide</h5>
                            <ul class="cat">
                                <li><i class="fa fa-angle-right"></i><a href="#navdescription">Description</a></li>
                                <li><i class="fa fa-angle-right"></i><a href="#navexample">Example</a></li>
                                <li><i class="fa fa-angle-right"></i><a href="#navarguments">Arguments</a></li>
                            </ul>
                        </div>
                        <div class="widget">
                            <h5 class="widgetheading">Download</h5>
                            <ul class="recent">
                                <li>
                                    <a href="https://github.com/shuangj00/HARMONIES/" class="btn btn-warning" style="width: 100%; text-align: center;"
                                       target="_blank">Link
                                        Github</a>
                                </li>
                            </ul>
                        </div>
                    </aside>
                </div>
            </div>
        </div>
    </section>
    <?php include("footer-small.php"); ?>
</div>
<a href="#" class="scrollup"><i class="fa fa-angle-up active"></i></a>
<!-- javascript
    ================================================== -->
<!-- Placed at the end of the document so the pages load faster -->
<script src="js/jquery.min.js"></script>
<script src="js/modernizr.custom.js"></script>
<script src="js/jquery.easing.1.3.js"></script>
<script src="js/bootstrap.min.js"></script>
<script src="js/jquery.appear.js"></script>
<script src="js/stellar.js"></script>
<script src="js/classie.js"></script>
<script src="js/uisearch.js"></script>
<script src="js/google-code-prettify/prettify.js"></script>
<script src="js/animate.js"></script>
<script src="js/custom.js"></script>

<script type="text/javascript">
    $("#navdownload").addClass("active");
</script>

</body>
</html>